packages/core/lib/console.js

```diff
@@ -46,6 +46,16 @@ const processInput = (input, allowedCommands) => {
     return makeIIFE(`ℹ️ : '${cmd}' is not a valid Truffle command`);
   }
 
+  if (words.includes("--url")) {
+    const message = "url option is not supported within Truffle REPL";
+    return makeIIFE(`ℹ️ : ${message}`);
+  }
+
+  if (words.includes("--network")) {
+    const message = "network option is not supported within Truffle REPL";
+    return makeIIFE(`ℹ️ : ${message}`);
+  }
+
```

Comment:
Let's add tests for these two

Reply:
I will add the tests. Thanks @cds-amal.

Reply:
The tests should cover both logic branches; with and without the `truffle` prefix.